source/extensions/quic_listeners/quiche/platform/quic_logging_impl.h

```diff
@@ -127,7 +127,7 @@ namespace quic {
 using QuicLogLevel = spdlog::level::level_enum;
 
 static const QuicLogLevel TRACE = spdlog::level::trace;
-static const QuicLogLevel DEBUG = spdlog::level::debug;
+static const QuicLogLevel QDEBUG = spdlog::level::debug;
```

Comment:
cc @danzh2010 David said you had a workaround for this - lmk if this conflicts or you have a better plan?  Honestly I was surprised this compiled, having assumed we needed to match the google log defines...

Reply:
cc @DavidSchinazi

Reply:
This works. I'll have a more complete fix which will bring back DEBUG (this time without breaking macOS) ASAP. Feel free to merge this.

Reply:
The other fix I mentioned is at https://github.com/envoyproxy/envoy/pull/15616